packages/editor/src/client/routes/userList.tsx

```diff
@@ -300,37 +279,7 @@ export function UserList() {
         />
       </Drawer>
 
-      <Modal show={isExportModalOpen} onHide={() => setExportModalOpen(false)}>
-        <Modal.Header>
-          <Modal.Title>{t('userList.panels.exportSubscription')}</Modal.Title>
-        </Modal.Header>
-
-        <Modal.Body>
-          <div style={{display: 'flex', justifyContent: 'flex-end'}}>
-            <Button appearance="primary" onClick={() => getSubsCsv()}>
-              {t('userList.panels.exportSubscription')}
-            </Button>
-          </div>
-          <Divider>{t('userList.panels.result')}</Divider>
-          <div style={{display: 'flex', width: '100%', justifyContent: 'flex-end'}}>
-            <div style={{wordBreak: 'break-word', marginRight: 10}}>
-              {isSubsLoading ? <Paragraph rows={6} /> : subsCsvData?.subscriptionAsCsv}
-            </div>
-            <IconButton
-              appearance="primary"
-              icon={<Icon size="lg" icon="copy" />}
-              disabled={!subsCsvData?.subscriptionAsCsv}
-              onClick={() => navigator.clipboard.writeText(subsCsvData!.subscriptionAsCsv!)}
-            />
-          </div>
-        </Modal.Body>
-
-        <Modal.Footer>
-          <Button onClick={() => setExportModalOpen(false)} appearance="default">
-            {t('userList.panels.close')}
-          </Button>
-        </Modal.Footer>
-      </Modal>
+      <SubscriptionAsCsvModal onHide={() => setExportModalOpen(false)} isOpen={isExportModalOpen} />
```

Comment:
I suggest you implement the `<subscriptionAsCsvModal>` component the same way as the the `<ResetUserPasswordPanel>` below.

Reply:
done